bin/setup.js

```diff
@@ -47,21 +47,39 @@ const questions = [
   },
   {
     name: "accessToken",
-    when: !argv.accessToken && !process.env.CONTENTFUL_ACCESS_TOKEN,
+    when:
+      !argv.accessToken &&
+      !process.env.CONTENTFUL_ACCESS_TOKEN &&
+      !argv.deliveryToken &&
+      !process.env.CONTENTFUL_DELIVERY_TOKEN,
     message: "Your Content Delivery API access token",
   },
 ];
 
 inquirer
   .prompt(questions)
   .then(({ spaceId, managementToken, accessToken }) => {
-    const { CONTENTFUL_SPACE_ID, CONTENTFUL_ACCESS_TOKEN } = process.env;
+    const {
+      CONTENTFUL_SPACE_ID,
+      CONTENTFUL_ACCESS_TOKEN,
+      CONTENTFUL_DELIVERY_TOKEN,
+    } = process.env;
 
     // env vars are given precedence followed by args provided to the setup
     // followed by input given to prompts displayed by the setup script
     spaceId = CONTENTFUL_SPACE_ID || argv.spaceId || spaceId;
     managementToken = argv.managementToken || managementToken;
-    accessToken = CONTENTFUL_ACCESS_TOKEN || argv.accessToken || accessToken;
+    // Some scripts that set up this repo use "deliveryToken" &&
+    // "CONTENTFUL_DELIVERY_TOKEN" instead of "accessToken" &&
+    // "CONTENTFUL_ACCESS_TOKEN". Until all scripts are updated to
+    // use "accessToken" && "CONTENTFUL_ACCESS_TOKEN" we will allow
+    // the deliveryToken variation to work.
```

Comment:
```suggestion
    // Some scripts that set up this repo use `deliveryToken` and
    // `CONTENTFUL_DELIVERY_TOKEN`, instead of `accessToken` and
    // `CONTENTFUL_ACCESS_TOKEN`. Until all scripts are updated to
    // use `accessToken` and `CONTENTFUL_ACCESS_TOKEN` both variations
    // will work.
```